File: snippets/java/array/reverse/Reverse.java
package array;

import java.util.Arrays;

public class Reverse {

    public static void main(String args[]) {
        int array[] = {1,2,3,4,5};
        for (int i = 0; i < array.length/2; i++) {
            int tmp = array[i];
            array[i] = array[array.length-(1+i)];
            array[array.length-(1+i)] = tmp;
        }
        System.out.println(Arrays.toString(array));

    }
}

File: snippets/cs/array/reverse/Reverse.cs

using System;
 
public class Reverse
{
    public static void Main()
    {
        int[] array = { 1, 2, 3, 4, 5 };
        Console.WriteLine("before reversing");

        foreach(int el in array){
            Console.WriteLine(el.ToString());
        }
        Array.Reverse(array);  // ≡

        Console.WriteLine("after reversing");

        foreach(int el in array){
            Console.WriteLine(el.ToString());
        }
    }
}
 